mod_reforged/hooks/msu.nut

```diff
@@ -8,6 +8,25 @@
 	return obj;
 }
 
+// Removes all duplicates from a string array and returns a new unique array
+::logInfo("Reforged::MSU -- adding ::MSU.Array.removeDuplicateStrings");
```

Comment:
This is just a "btw comment": This should ideally be `::MSU.Array.removeDuplicates` function which works with any data type. But it's ok for now as I don't think we will be keeping this long term.